### DISCUSSION TITLE
Additional reward for 3/3 challenge players

### DISCUSSION THREAD

**Discussion by trinitarius** [chromiecraft/chromiecraft#5702] [Suggestions]
First: **In my opinion the current rewards for the challenges are great**, a mount, pet, costume and T3 transmog are a good reward for doing one challenge for sure. Class specific rewards also enhance replayability of challenges.

However I would consider maybe adding some additional reward for those that complete all challenges together (3/3), as it is the most tedious and risky way of obtaining all the challenge rewards. Transmogs of Naxxramas weapons is a nice reward but the recent reveals of the class-specific rewards were questionable in my opinion. Problem is that opinions differ on the appearance of weapons and not everyone will be happy regardless which weapons are received. I would definitely keep Naxxramas weapons as reward, but here are different ideas I would discuss:

1. Give people the option to choose a weapon transmog or give all weapon transmogs at once. This would remove the class-specific reward of the weapon transmogs though which I am not in favour of. Class-specific rewards put more weight into certain sets because some classes are harder to play on 3/3.

2. Give people an additional reward that is beyond transmogrification. Transmog can be toggled off and so it is not seen by everyone. As stated the reward for 3/3 will be bragging rights so I think a reward that can brag better than a transmog item would be a good idea. As single challenges already reward mounts maybe a flying mount would be good. Maybe a title if there is the possibility to implent one.

I would like to hear about other people's opinions on this.

**Comment by Fluctisonant** [comment]
Hello Trinitarius,

I echo your sentiment with the 3/3 rewards not being up to par with the amount of time, effort, and to some extent, skill required to stay alive until 70 while undertaking all of the challenges.

I know this is a subjective and not an objective matter, but most of the classic Naxxramas weapons are extremely underwhelming and frankly ugly looking; far better weapon transmogs exist in the game that are far easier to obtain. I feel like there were around 2-3 people in the Discord asking about Naxxramas rewards being available for a while (I mostly took it as a meme) and somehow that became the centerpiece reward of the challenges. The Naxxramas armor transmog rewards are fine if you were to do the challenges separately or even attempt 2/3, but as it stands, the 3/3 bonus rewards are not worth chasing in any capacity. I truly realized that when I saw the 3/3 Warlock bonus rewards. There are no bragging rights with anything transmog related because as you mentioned, some people turn their transmogs off entirely.

I truly expected a unique flying mount or a title. That is a real bragging right and something worth working toward. As it stands now, 3/3 is not worth doing in any capacity. 

I've offered many suggestions and constructive criticism between two Discord servers but they've all fallen on deaf ears. I have a strong feeling your suggestion will as well, but at the very least we'll try! 😊✨🌸

**Reply by Amrx2** [reply]
Hi Fluctisonant, as with all suggestions we ask them to be placed here in the suggestions channel. Having hundreds if not thousands of messages to read on a daily basis means we cannot pick suggestions out, nothing about falling on deaf ears. It helps when stuff is posted in the right places so we can focus on them!

Regarding your suggestion about a unique flying mount - currently we have no way of creating a custom mount with flying stats what would work everywhere (like the Celestial Steed) and have it visible for all. By visible for all, meaning those who run the challenge mode patches and those who have the stock client (and not playing challenges).

The same issue exists for titles, yes we can create custom titles for those who do challenges but that means enforcing everybody to run custom patches in game to see them. If you don't have the patch, the player names aren't shown at all (or title, only the guild name is shown) and its important for us for players to play on CC with a clean 3.3.5a client. 

Regarding choosing a transmog weapon, yes that is something we can look into sorting for those which completed 3/3 or will complete in the future should there be the demand from the community from it. 

Cheers, 
Construct

**Reply by Mohdsaad1** [reply]
I think the ability to choose a weapon mog is the best solution at the moment, mainly because like you said, mount and title dont seem feasible.

At the moment, I think all 3 people who completed 3/3 would agree with this, and anyone who has substantially progressed in the triple challenge mode. It's 250+ hours of gameplay, it makes sense for the victor be satisfied with their accomplishment. 

As for weapon stats, I think its also likely that the majority of 3/3 players would agree that they would easily give up the weapon stats and 'bis'-ness for the ability to actually choose the weapon skin/mog, (if weapon stats becomes or is an issue).

As for 'falling on deaf ears', I think what Fluctisonant means is that its disheartening to not hear anything back over weeks of constant communication. Silence is worse than 'okay we're considering it' or just a 'you'll hear from us about this soon, we're thinking about' etc.

Sitting in silence sucks.

**Reply by Amrx2** [reply]
> Sitting in silence sucks.

I mean, its not like we haven't mentioned this place multiple times in the challenge channel: https://discord.com/channels/788063059926712341/1095827623973748767/1112440343988797461

We even have a dedicated suggestions channel what routes people here from our discord channel also. Anyway we can see somebody has finally made the suggestion so lets see what others have to say about this!

**Reply by Fluctisonant** [reply]
Hello Construct,

I appreciate you taking the time to address the suggestion here. My comment at the end of my message isn't meant to be vitriol. What I mean with "fallen on deaf ears" is that it's been quite a few weeks since Challenge Mode was released, and many participants including myself have expressed frustrations or shared suggestions between two Discords and the person in charge of the challenge (Roboto) has not participated in any of these conversations or even made the minimal effort to address any messages expressing frustration or suggestions at all. This isn't throwing sleight at Roboto or anything of the sort. I believe it to be valid criticism.

I am completely understanding of the client limitations and I am hopeful something can be figured out regarding the bonus rewards. It would surely motivate a lot more people to participate in the challenges, and I also strongly believe making the rewards public at this point is pertinent. The 'mystique' of what the rewards would be has long dissipated since many people have reached the finish line, and I've seen people in the Discord express disinterest when they're told that the rewards are not public and their hesitation to even bother participating in the challenges because of that is completely understandable.

If pets from later expansions that are offered as rewards such as the Holy Chime pet and so on can be implemented, could something like Reins of the Dark Phoenix be implemented as well? Maybe the bonus rewards for 3/3 can be given deeper consideration and thanks to your message here I'm certain you guys will figure it out.

Thank you for your contribution as well, Mohdsaad1!

Best, 
P

**Reply by Mohdsaad1** [reply]
I would argue the message you linked didnt do the trick of 'breaking the silence' because clearly people still feel like they werent heard, (and im not just referring to the person above).

https://discord.com/channels/788063059926712341/1095827623973748767/1114949438708916365

There was never a nod of 'oh yeah cool interesting idea we can think about it, post it in github in the meantime will ya?'

just a cold emotionless 'go github' sort of thing where it doesnt feel like the devs dont relate to what was being discussed at all in the challenges discord.
It sort of feels like we're being sent to the dump to be forgotten about? 
Just trying to get our feelings across, I dont hate anyone, I'm playing a great server for free and I'm happy about that.

and I understand you cant go through all the billion things going on in the disco, but the discussions been repeated over many days.

There arent many 3/3 enjoyers, and we're not going to be very loud just by the very nature of how few of us there are.

We dont ask for something that is difficult to script, or time consuming to fix.

**Comment by Mohdsaad1** [comment]
I completely agree the current 3/3 rewards that we've seen are not up to par, and its really not worth doing 3/3 for anything. No one will know you did 3/3 unless they go looking for your name on some random list in the corner of the city. You get nothing to show for it (what seem like almost randomized weapon mogs that will either be meaningless when wrath naxx launches, or mogs so ugly you wouldnt want to use them, or mogs that people wont see because they have tmog turned off).

1. I do believe a system where classes can choose from certain weapon mogs from naxx40 is best. Of course with certain class limitations (so not everyone is running hunter/lock as you mentioned). 


2. devs said cant implement titles because it would require everyone to use an mpq patch. And if you dont install it, names dont appear at all, so that probably can't work. As for mounts, I cant think of any that are both unobtaintable and also not on the CC store, so not sure there are any useable ones.

**Comment by ghost** [comment]
Hi there I would like to add a suggestion for an additional reward (I think the wep mogs should be kept and expanded to make all choices for that class available regardless of adding anything else)
As discussed earlier on discord, adding scarab mounts could be possible as an extra 3/3 unique mount reward since their models exist on the 3.3.5a client. This would not be the scarab lord already used by those who took part in the aq event, but the colored scarabs from the raid that are not normally usable outside the raid instance. There is a possibility of recoloring them or changing their size so they wouldn't be identical those you can ride in the raid, but either way this could be a unique mount as nobody but the scarab lords have scarabs out there, and a distinctly different color would draw eyes and provide the glory and fame that 3/3 pursuers deserve.
Considering the small pool of players who succeed with 3/3 I don't think this would anyhow devalue the existing scarab lords either.
Thanks for the consideration!

EDIT: if you can make the scarabs fly too people gonna drop their pants xD

PS: Different color of scarabs could correspond to different classes

**Reply by Mohdsaad1** [reply]
mount discussion aside, how would you distribute mog choices between the classes? I tried splitting weapons among classes with my limited naxx40 knowledge, feels pretty disastrous:

https://docs.google.com/spreadsheets/d/1HjTJnaTb_-CjsWXibkC0glvxPMyPoP24KnLoPxCaE5M/edit?usp=sharing

**Reply by ghost** [reply]
I think it should have to have overlaps. Offer each class 3-4 different weapons, some weapons would be given to multiple classes.

**Comment by TylerGoodguy** [comment]
Make Guard Thomas rideable as an exclusive mount for completing all challenges at the same time (3/3). The seated position of the player should be on his shoulders, as he will carry me through Azeroth. When the player learns how to use flying mounts (Expert Riding in TBC), Guard Thomas will perform HumanSwimmingAnimation02 and allow the player to fly as though they were using regular flying mounts.

Depending on the size of my character, this may cause me to be unable to go into certain sizes of entrances (Inns, Taverns, Bars, Pubs, etc.), but that is expected behavior with precedent in the game.

I appreciate your serious consideration of this unique opportunity to provide players with an exclusive and non-Blizzlike experience. This would only be available on ChromieCraft to those who have completed the most arduous goal of enlisting in the Ironman, Hardcore, and Bloodthirsty challenges.

**Comment by Never6** [comment]
Rewards are fine, and I don't think more need to be added but of course more is always better.
Just let people pick what rewards they want. If you are worried about everyone picking lock hunter than remove talent restriction cause for melee classes doing 3/3 is slow and tedious and takes a week to go from 60-70.

**Reply by GoblinKing1337** [reply]
if you make ironman talent restrictions it can become really easy (prot warr with talents in white gear still clean up crew). and it would suck for people who already completed ironman as is like boss without talents...

I think adding choice for the 3/3 items is best, limited choice for each class e.g hunter cannot pick corrupted ashbringer etc

**Reply by Never6** [reply]
Sure prot can do it easy with talents 
But hunter and lock can do it easy without them

**Reply by GoblinKing1337** [reply]
yeah but imagine how easy they will do with talent, any reward you associate with 3/3 everyone will get through hunter lock while sleeping

**Reply by Never6** [reply]
Ok than make exceptions cause the challenge is boring if you are doing it as a melee class
No gear no weps and missing key abilities
Like ret is judge auto till 70 nobody is gonna do that

**Reply by ghost** [reply]
It's kinda pointless to argue which class is easiest, or trying to make the challenge easier imho. The point is that its hard and tedious and it should be appropriately rewarded as such, and class specific rewards allow for incentive to complete challenge on more than 1 character

**Reply by Never6** [reply]
1v1ing a mob for 70 levels isn't a challenge, it is just tedious 
If entire challenge stems from playing in a boring way then it's not a challenge at all

**Comment by aradiuk** [comment]
I totally support the suggestion about letting players choose the desirable weapon transmog (I suggest not limiting it to Naxx only, though).
Also I think it would be great if we had a choice to pick the armor tmog (if I'm not mistaken it's t3 now, right?). I'd personally prefer the look of t1/t2 over t3.

Maybe as more content arrives new tmog 'options' would be unlocked.

**Comment by ghost** [comment]
Regarding 3/3 mount reward as a possibility. I've not seen the X-51 Nether-Rocket X-TREME on the shop. Unless I'm blind or it was on there in the past? It could be a candidate?

**Comment by balleny** [comment]
as getting a reward for HC being already accepted as the defacto standard: keep in mind on retail there are zero incentives besides the challenge itself. no rewards, especially not account-wide benefits

**Comment by ghost** [comment]
Found an NPC that could technically be turned into a mount. 
https://www.wowhead.com/npc=17536/nazan

Model not used by any other mount in the game. (obtainable by a player)
Not sure if it takes a lot of work to change an NPC into a mount though.

Just food for thought.
